src/canmatrix/__init__.py

```diff
@@ -9,7 +9,7 @@
 import canmatrix.formats as formats
 import canmatrix.cancluster as cancluster
 import canmatrix.convert as convert
-import canmatrix.copy as copy
+import canmatrix.cmcopy as cmcopy
```

Comment:
If anything is needed I believe `import canmatrix.copy` is sufficient.  But really if someone wants a submodule they should import it themselves, not expect importing it's parent to import it.